### PR TITLE
citadel: new keymap for pegasushoof

### PR DESCRIPTION
**Pull request by droseger**
This keymap is especially useful for users of ISO-DE or similar layouts.
It allows easy and ergonomic access to the AltGr (RALT) layer via CapsLock.

### CODE REVIEW

File: keyboards/pegasushoof/keymaps/citadel/readme.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+The _Function_ layer also offers mouse emulation and function/media keys in the F-Row.
+
+`Pause/Break` is used as a safe _close key_: A _single tap_ does nothing, a _double tap_ executes ALT+F4.
+
+Named after the homonymous space station from _Mass Effect_. Made by [Daniel Rose](https://github.com/droseger/).
+
+##[1] Layers
+
+###Layer 1 (Default)
```
Comment (drashna) [inline keyboards/pegasushoof/keymaps/citadel/readme.md]: Not formatted correctly.  Make sure that you have a double space at the end of each line.

File: keyboards/pegasushoof/keymaps/citadel/keymap.c [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "pegasushoof.h"
+
+/* Allow shortcuts for combos like RALT(KC_RBRC) that don't take up more than 7 characters: */
+#define G(kc) RALT(KC_##kc)
```
Comment (drashna) [inline keyboards/pegasushoof/keymaps/citadel/keymap.c]: That's clever! 👍

File: keyboards/pegasushoof/keymaps/citadel/readme.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    .----.   .-------------------. .-------------------. .-------------------. .--------------.
```
Comment (drashna) [inline keyboards/pegasushoof/keymaps/citadel/readme.md]: This isn't formatted correctly.  Make sure that you have a double space at the end of each line.

File: keyboards/pegasushoof/keymaps/citadel/readme.md (hunk 1 above)
Comment (droseger) [reply keyboards/pegasushoof/keymaps/citadel/readme.md]: I added some of those double spaces

**Comment by droseger** [comment]
I added double spaces yesterday, as you might have seen. Is there still anything that needs to be changed about my keymap? Thanks for reviewing.

**Comment by drashna** [comment]
Yeah, it's still having issues...  You can see what I mean here: https://github.com/droseger/qmk_firmware/blob/d1f51dfe5631342bbff08294fd1b7aa098164c4f/keyboards/pegasushoof/keymaps/citadel/readme.md

If you want, I can merge this still. Otherwise, I'd prefer to wait on you.  So, it's your call. Do you want to fix it, or should I just merge it now?

**Comment by noroadsleft** [comment]
@droseger All the headings (the lines that begin with one or more `#` characters) need to have a space after the `#` characters.

```
#Citadel keymap for pegasushoof  
```

should be
```
# Citadel keymap for pegasushoof
```

Do that and you don't need the two spaces at the end of the lines anymore, and it should render as desired.

\- @noroadsleft, self-described Markdown whisperer

**Comment by droseger** [comment]
I see what you mean. It looks completely perfect in CLion, so i figured you were talking about line breaks for the text. Good to know that CLion renders Markdown differently!

**Comment by drashna** [comment]
Looks good! 

Thanks!

**Pull request merged**